README.md

```diff
@@ -1,6 +1,202 @@
-# KubernetesAccessManager
-KubernetesAccessManager is a web application designed for multi-user management of Kubernetes cluster. It provides a graphical interface for users to manage their Kubernetes resources. Administrators can define precise access rights based on roles from an identity provider, such as Keycloak.
+<h1 style="display: flex; align-items: center; gap: 30px;">
+  Kubernetes Access Manager (KAM)
+  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
+</h1>
```

Comment:
It would be better to have logo above title, not in the same line.

Reply:
![Screenshot from 2024-12-10 23-20-46](https://github.com/user-attachments/assets/79665d6c-d412-4029-8050-4b0d71143e3f)
![Screenshot from 2024-12-10 23-21-08](https://github.com/user-attachments/assets/1ab4a2a1-2c28-49fd-a628-4cdd6725518b)
No

Reply:
@vgoriukhina przyjdź go wyjaśnij, @Nightbaka też mozesz, ale jedynie jeżeli się ze mna zgadzasz

Reply:
ŚRODEK